bluesky/run_engine.py

```diff
@@ -228,6 +228,18 @@ def __init__(self, md=None, *, loop=None, preprocessors=None,
         if md is None:
             md = {}
         self.md = md
+        self.md.setdefault('versions', {})
+
+        try:
+            import ophyd
+            self.md['versions']['ophyd'] = ophyd.__version__
+        except ImportError:
+            self.log.debug("Failed to import __version__ from ophyd.")
+
+        from ._version import get_versions
```

Comment:
Does not `import .__version__` work?

Reply:
import .__version__ doesn't seem to work.

Reply:
OK, thanks for the clarification. Never tried the versioneer from within a package.

Reply:
I think that would have to be `from . import __version__`. And this, as is, is probably slightly safer because it will avoid circular imports.

Reply:
Great, I like @danielballan's suggestion.

Reply:
@gwbischof, do you still plan to apply the suggested changes?

Reply:
Sorry, @mrakitin, I did not mean to mark this as resolved just now—that was a misclick.

But I think I phrased my suggestion poorly. I am suggesting to leave this as is because to import from the top level would create a circular import. To explain why: the top-level __init__.py imports from this already and with your change this would import from the top-level __init__.py. As written now there is no circular import.

Reply:
@danielballan, OK, it's a reasonable point.